src/main/resources/application.properties

```diff
@@ -35,4 +35,14 @@ spring.jpa.hibernate.use-new-id-generator-mappings=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.spatial.dialect.postgis.PostgisDialect
 
 # Liquibase Configuration
-liquibase.change-log=classpath:migrations/changelog.xml
\ No newline at end of file
+liquibase.change-log=classpath:migrations/changelog.xml
+
+# GeoServer Configuration
+geoserver.host=https://boundless-geoserver.dev.east.paas.nga.mil
```

Comment:
This needs to be removed. We will use pz-geoserver to pull the server host

We need to keep all customer addresses out of our source control.

Reply:
Looking at the properties in L2, the host isn't explicitly listed. Should we parse it from the url value?

Reply:
The VCAP property for GeoServer host in L2 is... unfortunate. Yes, we will have to do some manual parsing, likely in the Java code, to get the URL. You can use pz-access for an example on how we can do this.

Reply:
Done.